packages/angular/cli/utilities/color.ts

```diff
@@ -8,18 +8,16 @@
 import * as ansiColors from 'ansi-colors';
 import { WriteStream } from 'tty';
 
+type AnsiColors = typeof ansiColors;
+
 // Typings do not contain the function call (added in Node.js v9.9.0)
-export const supportsColor =
+const supportsColor =
   process.stdout instanceof WriteStream &&
   ((process.stdout as unknown) as { getColorDepth(): number }).getColorDepth() > 1;
 
-export function removeColor(text: string): string {
```

Comment:
This was added due to the `unstyle` method from the package not working when colors were disabled.
https://github.com/doowb/ansi-colors/blob/a4794363369d7b4d1872d248fc43a12761640d8e/index.js#L38